akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -95,11 +100,22 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
 
   logInfo("Starting up...")
 
+  @Deprecated @deprecated("Access via `clusterTeamInternalFailureDerector` instead", since = "2.5.4")
```

Comment:
to make it less confusing which one is which, once we settle on naming - would it be internalFailureDetector?